website/src/pages/documents/document.page.tsx

```diff
@@ -182,6 +182,10 @@ export const TranslationTab = ({ doc }: { doc: Document }) => {
     null
   )
 
+  useEffect(() => {
+    selectAndShowContent(null)
+  }, [doc.id])
+
```

Comment:
Do we have any preference for this over setting a `key` on this component, so that there is a full remount when the document id changes?

Reply:
Setting a key is probably good practice